frontend/src/redux/actions.js

```diff
@@ -73,27 +73,34 @@ export const setGoalListValue = (index, valueName, value) => ({
   payload: { index, valueName, value },
 });
 
-export const addGoal = () => ({
+export const addGoal = (goalFields) => ({
   type: "addGoal",
+  payload: { goalFields },
 });
 
 // finances
-export const setFinancesValue = (valueName, value) => ({
-  type: "setFinancesValue",
+export const setFinances = (valueName, value) => ({
```

Comment:
i think this one should have value at the end cuz its to change a general value in finances state